### ISSUE TITLE
Weird diff interpretation

### ISSUE THREAD

**Issue by vberger** [Ekleog/todiff#8]
Leaving here an example diff where todoff had a weird interpretation of what happened:

git diff:
```
-(D) 2018-03-22 Update lambda +root @computer rec:2w
+(D) 2018-04-01 Update lambda +root @computer rec:2w due:2018-04-15
+(B) 2018-04-01 Update PR platal#49 +x.org @computer
```

todiff output

```
No new tasks.

Changed tasks:
 → (D) 2018-03-22 Update lambda +root @computer rec:2w
    → Added due date 2018-04-15 and set creation date to 2018-04-01
    → Copied, set priority to (B), set creation date to 2018-04-01, removed tag rec:2w and set subject to ‘Update PR platal#49 +x.org @computer’
```

**Comment by Ekleog** [comment]
Just pushed ff30fc6 that should close this issue by allowing to set the similarity index: now, running `todiff a.txt b.txt --similarity=80` with your example correctly considers the diff as both a new task and a changed task.

Currently it's set at 50 by default, but that was finger-in-the-wind choice, so I'm open to suggestions, and think I'll likely switch over to something like 75 for the next release, as that should be enough to correctly detect tasks changing subject. This should be enough to default to a not-unexpected behaviour in most cases.

Does this sound good to you / works for you?

**Comment by Ekleog** [comment]
Just pushed an increase to 75 as 13369191a3866de318b8e4f0c2286a2d6a83af28 and released 0.4.0, so this issue should now be solved :)
